### PR TITLE
Ajiaco ranking

### PR DESCRIPTION
**Pull request by SamHadjes**

### CODE REVIEW

**Review by baaluidnrey — CHANGES_REQUESTED** [review]
I have the soup to try before the ranking !

**Comment by baaluidnrey** [comment]
forget my comment, it's not complete

File: README.md
Comment (baaluidnrey) [inline README.md:1]: I need to try this soup before the ranking :)

File: soups/ajiaco_soup.md
Comment (baaluidnrey) [inline soups/ajiaco_soup.md:1]: I really would like to try the receipe, How can I get these ingredients in France ?

**Review by baaluidnrey — CHANGES_REQUESTED** [review]
Thank you for your pull request, can you add some details ?

File: soups/ajiaco_soup.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Ajiaco
+*Recipe from : https://www.mycolombianrecipes.com/ajiaco-bogotano-colombian-chicken-and/*
+
+## Ingredients
+**Chicken:** I like to use bone in chicken breast to add more flavor to the soup, but feel free to use boneless if you prefer.
+
+**Water:** You can replace it for chicken or vegetable broth.
+
+**Herbs:** you need guascas to make ajiaco. I find them dried in small bags online here in the United States, but in Colombia you will find them fresh. I also like to add fresh cilantro to my soup, but if you don't like the flavor just omit it.
```
Comment (mstimberg) [inline soups/ajiaco_soup.md:9]:
```suggestion
**Herbs:** you need [guascas](https://blog.amigofoods.com/index.php/colombian-food/guascas/) to make ajiaco. I find them dried in small bags online here in the United States, but in Colombia you will find them fresh. I also like to add fresh cilantro to my soup, but if you don't like the flavor just omit it.
```

File: README.md [hunk 2]
```diff
@@ -12,3 +12,8 @@ some nice recipes.
 - [mains](mains)
 - [sides](sides)
 - [desserts](desserts)
+
+
+## Ranking of 5 best dishes
+- :one: [Ajiaco](soups/ajiaco_soup.md)
```
Comment (baaluidnrey) [inline README.md:18]:
```suggestion
- :one: [Ajiaco](soups/ajiaco_soup.md) (after the big coderefinery dishes competition)
```

**Review by baaluidnrey — COMMENTED** [review]
this is a suggestion

**Pull request merged**